test/unit/math/rev/functor/util_algebra_solver.hpp

```diff
@@ -180,16 +295,16 @@ inline void error_conditions_test(const F& f,
   std::vector<int> dat_int;
 
   std::stringstream err_msg;
-  err_msg << "algebra_solver: size of the algebraic system's output (2) "
+  err_msg << "size of the algebraic system's output (2) "
           << "and size of the vector of unknowns, x, (3) must match in size";
   std::string msg = err_msg.str();
   EXPECT_THROW_MSG(
-      algebra_solver_powell(non_square_eq_functor(), x, y, dat, dat_int),
+      general_algebra_solver(is_newton, non_square_eq_functor(), x, y, dat, dat_int),
```

Comment:
There was a comment asking if this should be general instead of just powell. I checked and newton currently gives a segfaults. I think the answer is yes (and so needs fixed), because I think someone could trigger this error with an algebra system Stan function that returned the wrong size thing.

Reply:
It looks like @SteveBronder fixed this below.